### PR TITLE
Added more boilerplate.

### PR DESCRIPTION
**Pull request by hdavidzhu**

### CODE REVIEW

**Comment by parisakianmajd** [comment]
npm start error: 
npm ERR! Darwin 15.4.0
npm ERR! argv "node" "/usr/local/bin/npm" "start"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3

npm ERR! missing script: start
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     https://github.com/npm/npm/issues

**Comment by zihualiu** [comment]
lgtm

**Comment by zihualiu** [comment]
lol

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>InstaFind</title>
+</head>
+<body>
+  <div id="app">
+    <pane-root></pane-root>
```
Comment (hdavidzhu) [inline dist/index.html:8]: What is `pane-root`?

**Comment by hdavidzhu** [comment]
LGTM

**Pull request merged**